src/main.rs

```diff
@@ -393,22 +398,33 @@ impl HabitCtl {
         if let Some(entry) = self.get_entry(&date, &habit.name) {
             if entry.value == "y" {
                 DayStatus::Done
+            } else if entry.value == "s" {
+                DayStatus::Skipped
             } else if self.habit_satisfied(habit, &date) {
                 DayStatus::Satisfied
+            } else if self.habit_skipified(habit, &date) {
+                DayStatus::Skipified
```

Comment:
That's a fun name :) It would probably be a good idea to document all the states somehow... that could be left for a followup, though!

Reply:
Yes, I have to admit, I had this as "satisficed" first and then changed it cause it sounded funnier. =p  
Done is to Satisfied  as 
Skip is to Skipified
but agree I am terrible at naming. 

Better suggestion?

Reply:
I'd say leave it in! A few comments explaining these would probably go a long way, though. :)